src/Command/GenerateSchemaCommand.php

```diff
@@ -57,98 +83,91 @@ protected function execute(InputInterface $input, OutputInterface $output)
                 $field['type'] = $this->parseType($field['roughType']);
-                if (false !== strpos('Required', $field['description'])) {
-                    $field['required'] = true;
-                }
-                $field['is_object'] = $this->isObject($field['type']);
-                $field['is_multiple_types'] = count(explode('|', $field['type'])) > 1;
-                $field['is_collection'] = strpos($field['type'], '[]') !== false;
+                $field['required'] = stripos('Required', $field['description']) !== false;
                 unset($field['roughType']);
             }
-            $object['parent'] = $this->getParent($object['name']);
-            
+            unset($field);
+
+            $type['parent'] = $this->getParent($type['name']);
         }
-        
+        unset($type);
+
         foreach ($methodNodes as $methodName => $method) {
             $tableNode = $method['tableNode'];
             $descriptions = $method['descriptions'];
             $parameters = [];
-            $tableNode->filter('tbody tr')->each(function (Crawler $rowNode, $rowNumber) use (&$parameters) {
-                if ($rowNumber === 0) {
-                    return;
-                }
-                
+            $tableNode->filter('tbody tr')->each(function (Crawler $rowNode) use (&$parameters) {
                 $type = $this->parseType($rowNode->filter('td:nth-child(2)')->text());
                 $description = $rowNode->filter('td:nth-child(4)')->text();
-                
+
                 $required = (false !== strpos($description, 'Required'));
 
                 if ($required === false) {
                     $required = $this->parseRequired($rowNode->filter('td:nth-child(3)')->text());
                 }
 
                 $parameters[] = [
-                    'name' => $rowNode->filter('td:nth-child(1)')->text(),
-                    'type' => $type,
-                    'is_multiple_types' => count(explode('|', $type)) > 1,
-                    'is_collection' => strpos($type, '[]') !== false,
-                    'is_object' => $this->isObject($type),
-                    'required' => $required,
+                    'name'        => $rowNode->filter('td:nth-child(1)')->text(),
+                    'type'        => $type,
+                    'required'    => $required,
                     'description' => $description,
                 ];
             });
 
             $description = implode("\n", $descriptions);
 
             $this->schema['methods'][] = [
-                'name' => $methodName,
-                'parameters' => $parameters,
+                'name'        => $methodName,
+                'parameters'  => $parameters,
+                'return'      => $this->getReturnType($description),
                 'description' => $description,
-                'link' => self::URL.$node->filter('a')->attr('href'),
+                'link'        => $node->filter('a')->attr('href'),
```

Comment:
The generated client is currently setting all the links to `#setwebhook`. I ran into this issue before and was able to fix it with:

```suggestion
                'link'        => self::URL.$tableNode->previousAll()->filter('h4 a')->first()->attr('href'),
```